lib/delegate/index.js

```diff
@@ -195,14 +196,14 @@ const createSubOperationDocument = function (component, targetRootField, args, s
 const mergeErrors = function (data, errors) {
```

Comment:
one other thing that popped in my mind is modifying the path in each error to include the path already traversed prior to calling `delegateToComponent`
ex: 
```graphql
query {
  foo {
    bar {
      a
      b
      c
     }
  }
}
```
If I delegate from bar for example and an error occurs on `a` - the error's path would only take into account the path starting at bar when I think the spec says wants a path starting at the root of the document. Just a thought

Reply:
+1, the delegate is just an internal mechanism of resolving types (by forwarding from parent to schema to another subschema), the path should start at root and end with the field associated with the error.